tedbottompicker/build.gradle

```diff
@@ -68,13 +55,17 @@ configurations {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     testCompile 'junit:junit:4.12'
-    compile 'com.android.support:appcompat-v7:24.2.0'
-    compile 'com.android.support:design:24.2.0'
+    compile 'com.android.support:appcompat-v7:25.0.1'
+    compile 'com.android.support:design:25.0.1'
 
-    compile 'com.android.support:support-annotations:24.2.0'
-    javadocDeps 'com.android.support:support-annotations:24.2.0'
+    compile 'com.android.support:support-annotations:25.0.1'
+    javadocDeps 'com.android.support:support-annotations:25.0.1'
     compile 'com.github.bumptech.glide:glide:3.6.1'
+    compile 'com.tbruyelle.rxpermissions:rxpermissions:0.7.0@aar'
```

Comment:
RxPermissions should be migrated to 0.9.1
https://github.com/tbruyelle/RxPermissions